.github/workflows/haskell.yml

```diff
@@ -36,7 +37,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.2.7", "9.6.4", "9.8.1"]
+        ghc: ["8.10.7", "9.6.4", "9.8.1"]
```

Comment:
We still use it as our main development ghc version. I'd prefer not to remove it from CI for now. Also `9.8.2` is now available
```suggestion
        ghc: ["8.10.7", "9.2.8", "9.6.4", "9.8.2"]
```

Reply:
We didn't move to 9.6? 👀

Reply:
@lehins I think we did move to `9.6`.

Reply:
Nope. Our main ghc for deployment of cardano-node is still 8.10.7

Reply:
But for development, it depends on the repo. For ledger it is still 9.2.8